Book-A-Desk.Domain/Office.Domain.fs

```diff
@@ -10,6 +10,7 @@ type Office =
         Id : OfficeId
         City : CityName
         BookableDesksPerDay : int
+        OpeningHoursText: string
```

Comment:
For now, this is just a string, we can rework it to something better once we implement the next story